inspirehep/modules/records/jsonschemas/records/hep.json

```diff
@@ -1,6 +1,11 @@
 {
     "$schema": "http://json-schema.org/schema#",
     "description": "An article or thesis or book or...",
+    "patternProperties": {
+        "^title\\.\\w{2}$": {
+            "$ref": "elements/title.json"
```

Comment:
Yep when I wrote this I also realized that the record-editor won't be happy as well. So probably we should delegate the difference to elasstiocsearch